### DISCUSSION TITLE
Unable to unbind ctrl+s keybind

### DISCUSSION THREAD

**Discussion by mock1328** [zellij-org/zellij#3640] [Q&A] [answered]
Have this in my `~/.config/zellij/config.kdl`:

```kdl
keybinds {
    unbind "Ctrl q"
    unbind "Ctrl s"
}
```

The first `unbind` works, the second doesn't

**Comment by Lockszmith-GH** [comment, marked as answer]
If I recall correctly, people were successful with specifying all the unbind keys in one line, like:
```kdl
keybinds {
    unbind "Ctrl q" "Ctrl s"
}
``` 
See: #1528

Zellij works in 'modes', with "Ctrl q" and "Ctrl s" defined in the normal mode. So a possible approach would be to unbind the keys within that mode:
```kdl
keybinds {
    normal {
        unbind "Ctrl q"
        unbind "Ctrl s"
    }
}
```
If this works, this might be a bug, as there was a case like this in the past.  
See: #2159

Another completely different approach would be to [dump the default](https://zellij.dev/documentation/configuration.html?highlight=dump#how-to-dump-the-default-configuration-to-stdout) and modifying it, adding `clear-defaults=true` and then modifying the setting with a 'blank slate' - no unbinding necessary.  
See: #1999

**Reply by mock1328** [reply]
Thanks, specifying with `normal` mode works like a charm!
